src/screens/Game/MathMain/MathMain.tsx

```diff
@@ -8,29 +8,36 @@ import Toast from "react-native-toast-message";
 import CustomButton from "../../../components/Button";
 import ProgressBar from "../../../components/ProgressBar";
 import Text from "../../../components/Text";
-import gameDescriptions from "../../Stacks/gameDescriptions";
-import useMathProblems from "../../../hooks/useMathProblems";
+import useMathProblems, { TOTAL_TIME } from "../../../hooks/useMathProblems";
 import { RemainingTimeGetter, RootStackParamList } from "../../../types";
 import styles from "./MathMain.style";
 
-const TOTAL_TIME = gameDescriptions.Math.minutes * 60;
 type Props = NativeStackScreenProps<RootStackParamList, "MathMain">;
 
 function MathMain({ route, navigation }: Props) {
   const [buttonsDisabled, setButtonsDisabled] = useState(false);
   const [skipped, setSkipped] = useState(false);
+  const firstTry = useRef(true);
```

Comment:
Move this to within the `useMathProblems` hook since this is only used internally to keep track of the number of questions answered correctly on first try. Update `firstTry.current` within `getNewProblem`, `updateStatsOnAnswer`, and `updateStatsOnSkip`.